test/DrawBBox.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python2
+import subprocess 
+import numpy
+import sys
+
+# Edit this as required (windows, mac).  Be able to write this.
+PLY_FILE="/tmp/displaz_bbox.ply"
+
+# write .ply file representing a box with specfied center and transform to f
+def MakeBBox(v_center, transform, f):
+  f.write("ply\n")
+  f.write("format ascii 1.0\n")
+  f.write("comment simple rectangular prism\n")
+  f.write("element vertex 8\n")
+  f.write("property float x\n")
+  f.write("property float y\n")
+  f.write("property float z\n")
+  f.write("element face 2\n")
+  f.write("property list uint int vertex_index\n")
+  f.write("end_header\n")
```

Comment:
Minor note - writing literal blocks of text like this to file can be done neatly with the standard `textwrap` module (see `textwrap.dedent` to remove pesky indentation from a big block of text in a function).